doc/ai-subnet.md

```diff
@@ -0,0 +1,533 @@
+    curl -O localhost:8936/stream/8794a01b/1f9bc7f2.mp4
+    ```
+
+## On-chain Configuration
+
+After successful **off-chain** testing of your Orchestrator and Gateway nodes, you can proceed with the **on-chain** configuration. This involves connecting your nodes to the [Livepeer protocol](https://livepeer.org/) for AI job processing on the _AI Subnet_. The instructions below will guide you through the steps to configure your Orchestrator and Gateway nodes **on-chain**, without affecting your main nodes.
+
+### On-chain AI Subnet Orchestrator Configuration
+
+To redeem _Mainnet AI Subnet_ tickets **on-chain**, ensure your _Mainnet Transcoding Network_ Orchestrator is set up and ranked in the top 100. Refer to the [Livepeer Orchestrator Setup Documentation](https://docs.livepeer.org/orchestrators/guides/mainnet-transcoding-network) for setup steps. Once set up, configure your _Mainnet AI Subnet_ Orchestrator **on-chain**. Run a separate Orchestrator for the _AI Subnet_ to avoid affecting your main Orchestrator. This Orchestrator will handle AI jobs on the _Mainnet AI Subnet_. If your main Orchestrator is operational, there are two methods for **on-chain** AI ticket redemption:
+
+-   **Method 1 (Recommended)**: Redeem the AI tickets **on-chain** on your _Mainnet AI Subnet_ Orchestrator, using the `-ethOrchAddr` to set your _Mainnet Transcoding Network_ Orchestrator as the tickets `recipient.
+-   **Method 2**: Set up a ticket redemption service using the `-redeemer` flag, and have your _Mainnet AI Subnet_ Orchestrator send the tickets to this service using the `redeemerAddr` flag.
+
+Below, you'll find comprehensive instructions for the first method. While a [second method is available](https://docs.livepeer.org/references/go-livepeer/cli-reference#onchain), it's generally not advised due to its requirement for configuring a ticket redemption service directly on your primary Orchestrator. This setup could impact the Orchestrator's performance, introducing extra latency and the possibility of other complications.
+
+#### Set Ticket Recipient
+
+The first and **recommended method** is to use the `ethOrchAddr` flag to set the Ethereum address of your _Mainnet Transcoding Network_ Orchestrator as the recipient of the AI tickets. This ensures that the AI tickets are redeemed **on-chain** by your _AI Subnet_ Orchestrator, while the AI rewards are sent to your _Mainnet Transcoding Network_ Orchestrator. Follow these steps to set up your _Mainnet AI Subnet_ Orchestrator using this method:
+
+1. Create a new Ethereum account for your _Mainnet AI Subnet_ Orchestrator. It is recommended that you use a separate account from your _Mainnet Transcoding Network_ Orchestrator for security reasons.
+2. Fund the Ethereum account with enough ETH to cover the gas costs of redeeming the AI tickets **on-chain**.
+3. Open port `8936` on your machine and set up port forwarding on your router to make the _Mainnet AI Subnet_ Orchestrator accessible from the internet.
+
```

Comment:
**Orchestrator running in WSL**: If you cannot access your orchestrator in port `8936` even after forwarding port in router and setting up firewall, use the [Netsh interface portproxy](https://learn.microsoft.com/en-us/windows/wsl/networking) guide from *Accessing a WSL 2 distribution from your local area network (LAN)*.

Reply:
@JJassonn69 thanks for the update. I will implement your changes in a subsequent pull request.